extensions/resteasy-reactive/jaxrs-client-reactive/deployment/src/main/java/io/quarkus/jaxrs/client/reactive/deployment/JaxrsClientReactiveProcessor.java

```diff
@@ -935,6 +939,44 @@ private void handleReturn(ClassInfo restClientInterface, String defaultMediaType
                 genericReturnType = createGenericTypeFromParameterizedType(methodCreator, paramType);
             }
         }
+        Integer continuationIndex = null;
+        //TODO: there should be an SPI for this
+        if (returnCategory == ReturnCategory.BLOCKING) {
+            try {
+                Thread.currentThread().getContextClassLoader().loadClass(UNI_KY.toString());
+            } catch (ClassNotFoundException e) {
+                //TODO: make this automatic somehow
+                throw new RuntimeException("Suspendable rest client method" + jandexMethod + " is present on class "
+                        + jandexMethod.declaringClass()
+                        + " however io.smallrye.reactive:mutiny-kotlin is not detected. Please add a dependency on this artifact.");
+            }
+            List<Type> parameters = jandexMethod.parameters();
+            for (int j = 0; j < parameters.size(); j++) {
```

Comment:
I don't think you need to loop here, I am almost certain that the `Continuation` is always the last parameter of the method